.github/workflows/test.yml

```diff
@@ -145,3 +145,9 @@ jobs:
         with:
           name: ocp-artifact
           path: ./ocp-artifacts
+
+      - name: remove virtual cluster
+        if: always()
+        run: |
+          kcli delete cluster $CLUSTER_NAME -y
+          kcli delete network $CLUSTER_NAME -y
```

Comment:
How can it ever have worked before if we need this now?

Reply:
it's already done but this is just to be on the safe side